pkg/account.go

```diff
@@ -461,15 +462,123 @@ func (c Client) UpdateAccount(account Account) (Account, error) {
 	return respAccount, ErrDefault(resp.StatusCode)
 }
 
+type AccountConfigurable func(c *Client)
+
+type AccountFilters struct {
+	// Name If provided, this query will attempt to find matches against the following Account and Profile fields: diplayName, firstName, middleName, lastName, legalBusinessName
+	Name string `url:"name"`
+	// Email Filter connected accounts by email address.
+	Email string `url:"email"`
+	// Type Possible values: individual, business
+	Type string `url:"type"`
+	// ForeignID Serves as an optional alias from a foreign/external system which can be used to reference this resource.
+	ForeignID string `url:"foreignID"`
+	// VerificationStatus Possible values: unverified, pending, resubmit, review, verified, failed
+	VerificationStatus string `url:"verification_status"`
+	// includeDisconnected If true, the response will include disconnected accounts.
+	IncludeDisconnected bool `url:"includeDisconnected"`
+	// Count Optional parameter to limit the number of results in the query. Default is 20
+	Count int `url:"count"`
```

Comment:
What do these `url` struct tags do?

Reply:
I can delete those. I was going to use this library at first and then I removed the struct with my lastest pull request 
https://github.com/google/go-querystring